packages/docusaurus/src/commands/build.ts

```diff
@@ -67,7 +67,17 @@ export async function build(
     ].filter(Boolean) as Plugin[],
   });
 
-  let serverConfig: Configuration = createServerConfig(props);
+  const staticDir = path.resolve(siteDir, 'static');
```

Comment:
Maybe shift 'static' into Docusaurus constants file here and below like we did for generated dir

Reply:
done